sources/packages/web/src/views/institution/locations/active-applications/ActiveApplicationReportAChange.vue

```diff
@@ -1,20 +1,25 @@
 <template>
-  <Card class="p-m-4">
-    <template #content>
-      <h1 class="d-flex justify-content-center">
-        Report a change - Work in Progress
-      </h1>
-      <formio
-        formName="reportachange"
-        :data="initialData"
-        @submitted="submitted"
-      ></formio>
-    </template>
-  </Card>
+  <v-card class="p-m-4">
+    <formio
+      formName="reportachange"
```

Comment:
Can the form name be more specific (e.g. reportscholasticstandingchange)?

Reply:
the form was already there, yes, i can rename the form

Reply:
Maybe wait to see if the others DEV agree with the name or came up with a better name. Either way, _reportachange_ is too generic IMO.

Reply:
Yea this was me when initially creating the form with Guru. From the institution side, report a change is specific enough. But @andrewsignori-aot  makes a good point. For us looking at the forms in formio, report a change is very generic. I would agree with renaming the form

Reply:
updated